monai/transforms/smooth_field/array.py

```diff
@@ -217,7 +325,137 @@ def __call__(self, img: np.ndarray, randomize: bool = True):
         field = self.sfield()
         rfield, *_ = convert_to_dst_type(field, img)
 
+        # everything below here is to be computed using the destination type (numpy, tensor, etc.)
+
         out = img * rfield
-        out, *_ = convert_to_dst_type(out, img, img.dtype)
 
         return out
+
+
+class RandSmoothDeform(RandomizableTransform):
```

Comment:
Could you please help add the transform image for these 2 new transforms?
https://github.com/Project-MONAI/MONAI/blob/dev/monai/transforms/utils_create_transform_ims.py#L705
If you don't want to investigate it, I can help add it later when this PR is merged.

Thanks in advance.

Reply:
I've added what I think should work for this, I've reduced the size of the random fields for all the smooth field transforms to (10,10,10) which should give better looking results.

Reply:
Thanks for your update, could you please help also upload the added / updated images to:
https://github.com/Project-MONAI/DocImages

Thanks in advance.

Reply:
I can update the images but maybe better if you could do it after merge as I haven't gotten into it yet. We should update the readme of the DocImages repo to describe what to do or automate it somehow.

Reply:
Sure, I can update that repo later.

Thanks.